src/khoj/processor/content/org_mode/org_to_entries.py

```diff
@@ -57,93 +54,175 @@ def process(
 
-            if parsed_entry.hasBody:
-                compiled += f"\n {parsed_entry.body}"
+        Each list of OrgNodes is a parsed parent org tree or leaf node.
+        Convert each list of these OrgNodes into a single Entry.
+        """
+        entries: List[Entry] = []
+        for entry_group in parsed_entries:
+            entry_heading, entry_compiled, entry_raw = "", "", ""
+            for parsed_entry in entry_group:
+                if not parsed_entry.hasBody and not index_heading_entries:
+                    # Ignore title notes i.e notes with just headings and empty body
+                    continue
+
+                todo_str = f"{parsed_entry.todo} " if parsed_entry.todo else ""
+
+                # Set base level to current org-node tree's root heading level
+                if not entry_heading and parsed_entry.level > 0:
+                    base_level = parsed_entry.level
+                # Indent entry by 1 heading level as ancestry is prepended as top level heading
+                heading = f"{'*' * (parsed_entry.level-base_level+2)} {todo_str}" if parsed_entry.level > 0 else ""
+                if parsed_entry.heading:
+                    heading += f"{parsed_entry.heading}."
+
+                # Prepend ancestor headings, filename as top heading to root parent entry for context
+                # Children nodes do not need ancestors trail as root parent node will have it
+                if not entry_heading:
+                    ancestors_trail = " / ".join(parsed_entry.ancestors) or Path(entry_to_file_map[parsed_entry])
+                    heading = f"* Path: {ancestors_trail}\n{heading}" if heading else f"* Path: {ancestors_trail}."
+
+                compiled = heading
                 if state.verbose > 2:
-                    logger.debug(f"Body: {parsed_entry.body}")
-
-            if compiled:
+                    logger.debug(f"Title: {heading}")
+
+                if parsed_entry.tags:
+                    tags_str = " ".join(parsed_entry.tags)
+                    compiled += f"\t {tags_str}."
+                    if state.verbose > 2:
+                        logger.debug(f"Tags: {tags_str}")
+
+                if parsed_entry.closed:
+                    compiled += f'\n Closed on {parsed_entry.closed.strftime("%Y-%m-%d")}.'
+                    if state.verbose > 2:
+                        logger.debug(f'Closed: {parsed_entry.closed.strftime("%Y-%m-%d")}')
+
+                if parsed_entry.scheduled:
+                    compiled += f'\n Scheduled for {parsed_entry.scheduled.strftime("%Y-%m-%d")}.'
+                    if state.verbose > 2:
+                        logger.debug(f'Scheduled: {parsed_entry.scheduled.strftime("%Y-%m-%d")}')
+
+                if parsed_entry.hasBody:
+                    compiled += f"\n {parsed_entry.body}"
+                    if state.verbose > 2:
+                        logger.debug(f"Body: {parsed_entry.body}")
+
```

Comment:
Do you think all these debug lines are useful to keep?

Is the reason you're explicitly adding the closed, scheduled, etc lines so that it can be preserved through all chunks of the entry?

Reply:
No, they're not required. They require extreme verbose mode to trigger (i.e `-vvv` flag). Removing it for now.

> Is the reason you're explicitly adding the closed, scheduled, etc lines so that it can be preserved through all chunks of the entry?

Yes, exactly.